src/components/ProfileList/index.js

```diff
@@ -1,19 +1,30 @@
-import Divider from '@material-ui/core/Divider';
 import React, { Component } from 'react';
-import ListItem from '@material-ui/core/ListItem';
 import Typography from '@material-ui/core/Typography';
 import Grid from '@material-ui/core/Grid';
-import Majors from './../../components/MajorList';
-import Minors from './../../components/MinorList';
 import user from './../../services/user';
-import Switch from '@material-ui/core/Switch';
 import Card from '@material-ui/core/Card';
 import CardHeader from '@material-ui/core/CardHeader';
 import CardContent from '@material-ui/core/CardContent';
-import LockIcon from '@material-ui/icons/Lock';
+import Snackbar from '@material-ui/core/Snackbar';
+import ErrorIcon from '@material-ui/icons/Error';
+import CloseIcon from '@material-ui/icons/Close';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import IconButton from '@material-ui/core/IconButton';
 import './profileList.css';
 import { withStyles } from '@material-ui/core/styles';
 import { gordonColors } from '../../theme';
+import {
+  createHomeListItem,
+  createHomePhoneListItem,
+  createMobilePhoneListItem,
+  createMajorsListItem,
+  createMinorsListItem,
+  createResidenceListItem,
+  createDormitoryListItem,
+  createFacultyDepartmentItem,
+  createMailboxItem,
+  createStudentIDItem,
```

Comment:
These buttons used to be in this file, they have been placed in a separate file and are imported here to keep the code clean and small as possible